packages/cli/src/aws/update-app.test.ts

```diff
@@ -197,6 +197,65 @@ describe('update-app command', () => {
     expect(console.log).toBeCalledWith('Done');
   });
 
+  test('Update app command without optional configs', async () => {
```

Comment:
Adding test to cover: https://sonarcloud.io/code?id=medplum_medplum&pullRequest=1944&selected=medplum_medplum%3Apackages%2Fcli%2Fsrc%2Faws%2Fupdate-app.ts&line=39